notebooks/tel_and_site/subsys_req_ver/m1m3/SITCOM-1136_M1M3_analysis_stability_tracking.ipynb

```diff
@@ -0,0 +1,704 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    count_is_unstable = {"True": sum(event["is_unstable"] for event in unstable_events.values()),
</pre><p> </p><p>Here you can also add some comments on the results (for the default dayObs you selected) you obtain. That will help to guide user to run the notebook for other dayObs, if needed.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_vandv/pull/107/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Reply:
<p>I have added some comments</p>